test/screenshotter/ss_data.yaml

```diff
@@ -49,9 +49,10 @@ Cases: |
         a-1&\text{otherwise}
     \end{cases}
 Colors:
-    tex: \blue{a}\color{#0f0}{b}\color{red}{c}
+    tex: \blue{a}\textcolor{#0f0}{b}\textcolor{red}{c}
     nolatex: different syntax and different scope
-ColorSpacing: \color{red}{\displaystyle \int x} + 1
+ColorSpacing: \textcolor{red}{\displaystyle \int x} + 1
+ColorImplicit: \color{red}red\color{blue}blue
```

Comment:
I think this test should ensure that a) ending a group restores previous color and b) ending a `\textcolor` does return to the previous implicit color. Something like

```tex
bl{ack\color{red}red\textcolor{green}{green}red\color{blue}blue}black
```